doc/migration/4.x-5.0.md

```diff
@@ -6,9 +6,11 @@
 
 2. Run the following script to export the currently available EvaluationSeasons: [DumpEvaluationSeason](https://gist.github.com/pedrosan7os/53ff0165cf13266857f9). Keep the generated file.
 
+3. Run the following script to export the currently available StatuteTypes: [StatuteTypes](https://gist.github.com/NunoPinheiro/9fd5bb667e8b1d77de03). Keep the generated file.
```

Comment:
The generated sql has syntax errors, json parts like {"pt-PT":"Finalista"} should be around '